JMS/README.md

```diff
@@ -372,7 +372,7 @@ The samples also contain unit tests in `src/test`. These tests require connectio
 To run these tests, Use the following command:
 
 ````
-mvn test
+mvn -Dmaven.test.skip=false test
```

Comment:
While we are here, we should update references to the knowledge centre to the latest version. See line 357 of this readme - the link should be updated to latest - ie (https://www.ibm.com/support/knowledgecenter/SSFKSJ_9.1.0/com.ibm.mq.dev.doc/q113220_.htm)
becomes 
https://www.ibm.com/docs/en/ibm-mq/latest?topic=jms-tls-cipherspecs-ciphersuites-in-mq-classes

Reply:
Still pointing at old documentation - please update.